Client/Frontend/Browser/Toolbars/BottomToolbar/Menu/DownloadsViewController.swift

```diff
@@ -295,13 +296,6 @@ class DownloadsPanel: UIViewController, UITableViewDelegate, UITableViewDataSour
         return configureDownloadedFile(cell, for: indexPath)
     }
     
-    func tableView(_ tableView: UITableView, willDisplayHeaderView view: UIView, forSection section: Int) {
-        if let header = view as? UITableViewHeaderFooterView {
-            header.textLabel?.textColor = UIColor.Photon.Grey90
-            header.contentView.backgroundColor = UIColor.Photon.Grey10
-        }
-    }
```

Comment:
@iccub There was a bug that was messing up the text/background color when dismissing the preview. Let me know if this code is still needed for something else

Reply:
It's ok to remove, it was a copy pasta from FF